reference/debugging.md

```diff
@@ -0,0 +1,300 @@
+# Debugging
+
+If you’re experiencing issues with Ambassador and cannot diagnose the issue through the "Diagnostics" tab from the [Edge Policy Console](../../about/edge-policy-console), this document covers various approaches and advanced use cases for debugging Ambassador issues.
+
+First, create an example configuration for debugging demonstrations that are available throughout this document.
+
+Then, complete the following debugging options:
+
+* Check Ambassador Status
+* Review Ambassador Logs
+* Examine Pod and Container Contents
+* Customize the Envoy Configuration
+
+## Example Configuration
```

Comment:
It seems weird to dedicate this much of the page to setting up a cluster to debug.  If you're interested in this page, you almost certainly already have a cluster with Ambassador installed.

Reply:
what do you recommend for this?

Reply:
I'm not sure.  I figure this page is largely born out of conversations that @nbkrause and @kflynn have hade with users: What do they think is a reasonable "set-up" for this?

Reply:
It looks like this is the only CRD being used:

```yaml
---
apiVersion: getambassador.io/v2
kind:  Mapping
metadata:
  name:  httpbin
spec:
  prefix: /httpbin/
  service: httpbin.org
  host_rewrite: httpbin.org
```

Reply:
(All the stuff from your "?" above and before the "Check Ambassador Status" below should go.)